src/app/util/static-data.js

```diff
@@ -56,7 +56,28 @@ const AMOUNTS_MAGIC_NUMBERS = [
   {
     label: '3 Items with delivery and service charge (if merchant supports it)',
     id: '62'
-  }
+  },
+  { label: '1 Item -> after 30 seconds: updated to show the item refunded', id: '70' },
+  { label: '3 Items -> after 30 seconds: updated to show a single item refunded', id: '71' },
+  { label: '3 Items -> after 30 seconds: updated to show all items refunded', id: '72' },
+  {
+    label: '3 Items with tax -> after 30 seconds: updated to show a single item refunded',
+    id: '73'
+  },
+  {
+    label: '1 Item with quantity = 3 -> after 30 seconds: updated with the item (quantity = 1) refunded',
+    id: '74'
+  },
+  {
+    label: '1 Item with tax and quantity = 3 -> after 30 seconds: updated with the item (quantity = 1) refunded',
+    id: '75'
+  },
+  {
+    label: '1 Item with quantity = 3 -> after 60 seconds: updated with the item (quantity = 3) refunded',
+    id: '76'
+  },
+  { label: '1 Item that has been refunded to the customer', id: '80' },
+  { label: '1 Item with tax that has been refunded to the customer', id: '81' }
```

Comment:
Is the idea that these just generate static return receipts?

Reply:
yeah. Just with a single item that has been returned. Maybe we should have one with the returned item and an exchanged item... thoughts?